### PR TITLE
added new css file

### PR DESCRIPTION
**Pull request by Holardeeps**
check the file

### CODE REVIEW

**Comment by amorgrace** [comment]
i never see the code

File: css/new.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.{
+    border: 0;
+    border: 0;
+    border: 0;
+    border: 0;
```
Comment (amorgrace) [inline css/new.css:5]: i no like this borderline here

**Review by amorgrace — APPROVED** [review]

File: css/new.css (hunk 1 above)
Comment (Holardeeps) [reply css/new.css:5]: cool

File: css/new.css [hunk 2]
```diff
@@ -0,0 +1,12 @@
+.{
+    border: 0;
+    border: 0;
+    border: 0;
```
Comment (amorgrace) [inline css/new.css:4]: nice layout change this icon

**Review by amorgrace — CHANGES_REQUESTED** [review]

**Pull request merged**